ports/espressif/common-hal/paralleldisplay/ParallelBus.h

```diff
@@ -32,8 +32,8 @@
 
 typedef struct {
     mp_obj_base_t base;
-    uint8_t read_pin_number;
-    uint8_t reset_pin_number;
+    int8_t read_pin_number;
+    int8_t reset_pin_number;
```

Comment:
You are right about compatibility with the IDF. But If you look at
https://github.com/adafruit/circuitpython/blob/08b44eade5f3caa25b986524d80ca6da2313bf20/ports/espressif/common-hal/imagecapture/ParallelImageCapture.h#L36-L38, it uses `gpio_num_t` for a similar purpose, and that's the actual enum type. So that's what I'd suggest here. That ends up being a 32-bit int, but that's OK. It could also use an `mcu_pin_obj_t *`, but that may not be an improvement.

```suggestion
    gpio_num_t read_pin_number;
    gpio_num_t reset_pin_number;
```